conference-application/agenda-service/agenda-service.go

```diff
@@ -8,100 +8,100 @@ import (
 	"net/http"
 	"os"
 
-	"math/rand"
-
+	"github.com/go-chi/chi/middleware"
+	"github.com/go-chi/chi/v5"
 	"github.com/google/uuid"
-	"github.com/gorilla/mux"
 	"github.com/redis/go-redis/v9"
 	kafka "github.com/segmentio/kafka-go"
-	"golang.org/x/exp/slices"
+)
+
+var (
+	KEY                 = "AGENDAITEMS"
+	VERSION             = getEnv("VERSION", "1.0.0")
+	SOURCE              = getEnv("SOURCE", "https://github.com/salaboy/platforms-on-k8s/tree/main/conference-application/agenda-service")
+	POD_NAME            = getEnv("POD_NAME", "N/A")
+	POD_NAMESPACE       = getEnv("POD_NAMESPACE", "N/A")
+	POD_NODENAME        = getEnv("POD_NODENAME", "N/A")
+	POD_IP              = getEnv("POD_IP", "N/A")
+	POD_SERVICE_ACCOUNT = getEnv("POD_SERVICE_ACCOUNT", "N/A")
+	REDIS_HOST          = getEnv("REDIS_HOST", "localhost")
+	REDIS_PORT          = getEnv("REDIS_PORT", "6379")
+	REDIS_PASSWORD      = getEnv("REDIS_PASSWORD", "")
+	KAFKA_URL           = getEnv("KAFKA_URL", "localhost:9094")
+	KAFKA_TOPIC         = getEnv("KAFKA_TOPIC", "events-topic")
+)
+
+const (
+	ApplicationJson = "application/json"
+	ContentType     = "Content-Type"
 )
 
 type Proposal struct {
-	Id string
+	Id string `json:"id"`
 }
 
 type AgendaItem struct {
```

Comment:
@mcruzdev do we still need these? because these should be auto generated from the openapi schemas right?